### ISSUE TITLE
Global terminal

### ISSUE THREAD

**Issue by topliceanurazvan** [kubeshop/monokle#3323]
Currently, access to the terminal is limited to the editor view (PaneManager), and any terminal processes running are lost upon navigating to the start page, as the terminal(s) is/are remounted. To improve the user experience, the terminal should be displayed persistently at the bottom of all pages (including edit and start pages) to allow users to interact with it continuously until intentionally closed.

**Comment by chargio** [comment]
Agreed. The terminal should be global, and when you open a terminal for a pod or other, it should create a new tab. Once you navigate away, it should be possible to keep it open.
Setting: "Automatically close the terminal when a resource is no longer selected YES/NO"
